### DISCUSSION TITLE
Multiple Orders & those orders active state

### DISCUSSION THREAD

**Discussion by TLable** [MicroTrendsLtd/NinjaTrader8#68] [General]
What is the mthod that uses the IsActivecanCancel and also allows to cancel the stop order, so that I can update to a new price, and then still keep when canceling those OCO order Targets. Can I append to the cancel to only exit the orders that are stops, if so what function or method can cancelorder work on a specific base.orderstop# aswell keeping that stops OCO target order. If that is not clear let me know I will answer any Q's ab above statement.

**Comment by MicroTrendsTom** [comment]
you can read the sample code in the method provided

e.g:        public override void TradeManagement(double lastPrice)
https://github.com/MicroTrendsLtd/NinjaTrader8/blob/main/ATSQuadroStrategyBase/NinjaTrader%208/bin/Custom/Strategies/ATSSamplePriceReversalTest.cs
